manifests/t/Telegram/TelegramDesktop/4.2.0/Telegram.TelegramDesktop.locale.en-US.yaml

```
@@ -0,0 +1,24 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: Telegram.TelegramDesktop
+PackageVersion: 4.2.0
```

```suggestion
PackageVersion: 4.2
```
This must match Control Panel.